### PR TITLE
Lazy loading blog post

### PR DESCRIPTION
**Pull request by rviscomi**
cc @felixarntz

### CODE REVIEW

File: src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+tags:
+  - blog
+  - web-vitals
+  - performance
+  - images
+---
+
+Lazy-loading is a technique to defer downloading a resource until it's needed, which conserves data
+and reduces network contention for critical assets. It became a web standard in
+[2019](https://web.dev/browser-level-image-lazy-loading/) and today `loading="lazy"` for images is
```
Comment (rachelandrew) [inline src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md:20]:
```suggestion
[2019](/browser-level-image-lazy-loading/) and today `loading="lazy"` for images is
```

File: src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+[rate of adoption](https://gist.github.com/rviscomi/44d80c1a0f4dec9cbafb37347c770278#file-lazy-loading-wp-cms-timeseries-sql)
+is also worth noting. One year ago in July 2020, WordPress sites that use lazy-loading made up tens
+of thousands websites in the corpus of about 6 million (1% of total). Lazy-loading adoption in
+WordPress alone has since grown to over 1 million websites (14% of total).
+
+## Correlational performance
+
+[Digging deeper](https://gist.github.com/rviscomi/44d80c1a0f4dec9cbafb37347c770278#file-lazy-loading-crux-lcp-sql)
+into HTTP Archive, we can compare how pages with and without native image lazy loading perform with
+the [Largest Contentful Paint](https://web.dev/lcp/) (LCP) metric. The LCP data comes from real user
```
Comment (rachelandrew) [inline src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md:83]:
```suggestion
the [Largest Contentful Paint](/lcp/) (LCP) metric. The LCP data comes from real user
```

File: src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+
+To summarize the results of the A/B test, the lazy-loading technique used by WordPress very clearly
+helps reduce image bytes but at the cost of a delayed LCP.
+
+## Testing a fix
+
+Before we get into how the fix was implemented, let's look at how lazy-loading works in WordPress
+today. The most important aspect of the current implementation is that it lazy-loads images above
+the fold (within the viewport). The CMS blog post
+[acknowledges](https://web.dev/browser-level-lazy-loading-for-cmss/#avoid-lazy-loading-above-the-fold-elements)
```
Comment (rachelandrew) [inline src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md:252]:
```suggestion
[acknowledges](/browser-level-lazy-loading-for-cmss/#avoid-lazy-loading-above-the-fold-elements)
```

File: src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+heuristics, especially the viewport size and the usage of anchor links that change the scroll
+position of the page. For those reasons, it's important to acknowledge that these results may not be
+universally applicable to all real-world scenarios and need more fine-tuning.
+
+## Rolling it out
+
+Now that we've identified a better way to lazy-load images, all of the image savings and faster LCP
+performance, how can we get sites to start using it? The highest priority change is to submit a
+patch to WordPress core to implement the experimental fix. We'll also be updating the guidance in
+the [Browser-level lazy-loading for CMSs](https://web.dev/browser-level-lazy-loading-for-cmss/) blog
```
Comment (rachelandrew) [inline src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md:391]:
```suggestion
the [Browser-level lazy-loading for CMSs](/browser-level-lazy-loading-for-cmss/) blog
```

File: src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md [hunk 5]
```diff
@@ -0,0 +1,431 @@
+---
+title: Lazy-loading is a prickly pear
+subhead: Optimizing for performance without the sharp edges.
+description: Recent research has shown that lazy loading images above-the-fold can lead to regressions in Core Web Vitals performance.
+authors:
+  - rviscomi
+  - felixarntz
+date: 2021-07-14
+hero: image/STd8eW8CSiNp5B1bX0R6Dww2eH32/h5OlDz9eOONtAi8JNqh7.jpg
```
Comment (rachelandrew) [inline src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md:9]: this image doesn't look great in the preview.

Comment (rviscomi) [reply src/site/content/en/blog/lazy-loading-is-a-prickly-pear/index.md]: Reuploaded a higher res image

Also ironically web.dev lazy loads the hero image!! 😆

**Comment by robdodson** [comment]
In general LGTM but I asked Rick if he could find another hero image as this one has issues on larger displays.

File: src/site/content/en/blog/lcp-lazy-loading/index.md [hunk 6]
```diff
@@ -0,0 +1,436 @@
+---
+title: The performance effects of too much lazy-loading
+subhead: Data-driven advice for lazy-loading images with Core Web Vitals in mind.
+description: "Eagerly loading images within the initial viewport—while liberally lazy-loading the rest—can improve Web Vitals while loading fewer bytes."
+authors:
+  - rviscomi
+  - felixarntz
+date: 2021-07-15
+hero: image/STd8eW8CSiNp5B1bX0R6Dww2eH32/Hdl5V6rz7DOLmC7LkYaX.jpg
+alt: Photograph of an opuntia cactus leaf growing, by ZooFari via Wikipedia.
```
Comment (robdodson) [inline src/site/content/en/blog/lcp-lazy-loading/index.md:10]: nit, need to update the photo alt

**Review by robdodson — APPROVED** [review]

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**